### DISCUSSION TITLE
TimeoutError: Navigation timeout of 30000 ms exceeded

### DISCUSSION THREAD

**Discussion by josecasique** [spatie/browsershot#723] [Bugs]
Hi,

I have a problem generating a PDF using Browsershot, the error is generated **TimeoutError: Navigation timeout of 30000 ms exceeded** in my local env.

My controller is like this

`Browsershot::html($html)
                            ->setNodeBinary('/usr/bin/node')
                            ->setNpmBinary('/usr/bin/npm')
                            ->setOption('args', ['--disable-web-security'])
                            ->showBackground()
                            ->showBrowserHeaderAndFooter()
                            ->hideHeader()
                            ->footerHtml(view('pdfs.footer')->render())
                            ->margins(30,10,30,10)
                            ->scale(0.9)
                            ->noSandbox()
                            ->emulateMedia('screen')
                            ->waitUntilNetworkIdle()
                            ->ignoreHttpsErrors()
                            ->setChromePath("/usr/bin/google-chrome")
                            ->addChromiumArguments([
                                'font-render-hinting' => 'none',
                            ])
                            ->savePdf(storage_path("app/reports/$file"));`

I'm using Laravel Sail in Mac OS, and building the container using FROM ubuntu:22.04/amd64

I increase the timeOut option to 3 minutes and the error persist. When checking running processes can see Chrome start then Chrome_chrashpad also starts but can't see what error occurs.

Any suggestion .

**Comment by Chitzi** [comment]
Same here, it seems that it can't recoignize when is idle. Or maybe it never will.

**Comment by vinnygambiny** [comment]
I have the same error, do you figured out ?
